src/actions/admin.js

```diff
@@ -380,17 +380,43 @@ export function getPlugins(): ActionFunc {
     };
 }
 
+// EXPERIMENTAL - SUBJECT TO CHANGE
+export function getPluginStatuses(): ActionFunc {
```

Comment:
Is there any reason to not use `bindClientFunc`?

Reply:
Hah! I hadn't looked earlier in the file to see that pattern.

I can use it here, but without modifying the helper, I'll need to continue with this pattern so that I can pass the `pluginId` affected by the other actions.

Reply:
This https://github.com/mattermost/mattermost-redux/pull/476/files#diff-11fe928fa6662326b67b6a3e602b987dR1286 was the solution from @mkraft to that problem.